src/main/kotlin/org/jitsi/jibri/sipgateway/SipClient.kt

```diff
@@ -34,7 +34,17 @@ data class SipClientParams(
      * Whether auto-answer is enabled, if it is, the client will listen for
      * incoming invites and will auto answer the first one.
      */
-    val autoAnswer: Boolean = false
+    val autoAnswer: Boolean = false,
```

Comment:
Are we planning on supporting the non auto-answer case?

Reply:
This is the xmpp side triggering this, as it creates a sip call.